UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Http;
```

Comment:
Please remove the unnecessary using statements. At line 1, 3-6. Also remove the unnecessary using statements from SalesCatalogueServiceController.cs file.

Reply:
Yes removed unused using statements